maquetado/setup/scss/layout.scss

```diff
@@ -1,9 +1,14 @@
-.techs-containers {
+.content {
+  margin-left: 80px;
+  margin-top: 150px;
```

Comment:
`margin: top right bottom left` => `margin: 150px 0 0 80px`

Although I would do `margin: 150px 80px 0` that will be `top horizontal bottom`